test/Libraries/DynamoMSOfficeTests/ExcelTests.cs

```diff
@@ -623,7 +623,7 @@ public void CanAddNullItemToExcelWorksheet()
             Assert.IsNull(watch.CachedValue.Data);
         }
 
-        [Test, Category("ExcelTest")]
+        [Test, Category("ExcelTestInterop")]
```

Comment:
instead of changing the category, can you add more categories - theres likely some ci scripts that rely on this.

Reply:
I've added the ExcelTest category then now those tests will have both Categories.
commit: https://github.com/DynamoDS/Dynamo/pull/15743/commits/e4af9e6052b108d69aa9ea99bafa4da868670cce